src/components/session-row.tsx

```
@@ -0,0 +1,131 @@
+import { ArrowRotateRight, FolderOpen, TrashBin } from '@gravity-ui/icons'
+import { Button, Checkbox, Chip, Spinner } from '@heroui/react'
+import { useTranslation } from 'react-i18next'
+import { If } from 'react-if-lite'
+import type { SessionFileInfo } from '@/hooks/use-dsh-sessions'
```

ESLint

[error] Expected "`@/hooks/use-dsh-sessions`" (type-internal) to come before "react-if-lite" (value-external).

(perfectionist/sort-imports)